README.md

```diff
@@ -160,6 +160,11 @@ The map of containers consists of the name of the container mapped to the contai
 * `build` (object, optional): Parameters mapped to Docker's `build`.
 	* `context` (string)
 	* `file` (string)
+* `push` (object, optional): Parameters related to Docker's `push`.
+  * `skip` (boolean): prevents pushing image when `push` is called
```

Comment:
I understand. My comment was the process will look like:
```
$ crane build platform
$ crane push platform-push
```
This flow isn't clear enough and bug prone. We have few cases when we wish to skip a specific container push (but not the main scenario).